Gaijinizer patch/Map106.txt

```diff
@@ -49,6 +49,6 @@ Demon Realm! Because I put in the item of
 > CONTEXT: Map109/events/109/pages/2/5/Dialogue/hyakki
 > CONTEXT: Map110/events/110/pages/1/5/Dialogue/hyakki
 > CONTEXT: Map110/events/110/pages/2/5/Dialogue/hyakki
-If people say that they failed to fortify with
-capsules. It may be useful!!#MTLed
+They might be useful for those people who've
+failed to strengthen with the capsule!!#Ron
 > END STRING
```

Comment:
This set of lines is specifically referring to the Charred Salamander. Because of this, I'd like to alter it a tiny bit...
>_**They might be useful for those people who've
failed to strengthen properly with the capsule!!**_